roles/sriov/defaults/main.yml

```diff
@@ -7,7 +7,7 @@ sriov_mft_url: https://www.mellanox.com/downloads/MFT/mft-4.23.0-104-x86_64-rpm.
 
 sriov_mft_tarball: "{{ sriov_mft_url | urlsplit('path') | basename }}"
 
-sriov_os_pkgs:
+sriov_os_pkgs_rhel:
```

Comment:
if you name the other deb, then name this rpm?

Reply:
Makes sense, done